main.go

```diff
@@ -174,3 +174,7 @@ func openFile(filename string) (*os.File, error) {
 
 	return fileToWrite, nil
 }
+
+func intToString(x int) (string) {
```

Comment:
Single return types don't need to be wrapped in parenthesis.

Can instead be 
```suggestion
func intToString(x int) string {
```